pkg/pillar/types/volumetypes.go

```diff
@@ -119,6 +119,12 @@ func (status VolumeStatus) IsContainer() bool {
 	return false
 }
 
+// VolumeName returns the name of the volume
+func (status VolumeStatus) VolumeName() string {
```

Comment:
Pathname would be a more accurate name for this function.

Reply:
Resolved.